### PR TITLE
The Great Conflictening 2.0: Removes /obj/item/device

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Removes `/obj/item/device`

@Crazylemon64 recommended doing this as well, and I agree with it. Pushed for this over at TG and it went through--we should follow up, as well.

This is another redundant path that serves virtually no purpose; the only time it's ever been used is for the wizard animation spell; objects that were `/obj/item/device` would be electronic. It's not worth keeping the path just for that.

This is yet another path that's just bloat, much like `/obj/item/weapon`.

This should largely kill off 99% of the redundant useless type-paths after this PR gets merged.

Another "soon rather than later" PR for merging.

### CODE REVIEW

File: code/modules/pda/radio.dm [hunk 1]
```diff
@@ -1,9 +1,9 @@
-/obj/item/radio/integrated
+/obj/item/integrated_radio
```
Comment (Fox-McCloud) [inline code/modules/pda/radio.dm]: I reserve a special place in hell for this object.

Someone, way back in the day, device to make `/obj/item/radio/integrated`, and make it distinctly different from `/obj/item/device/radio` The worst part? `/obj/item/radio` doesn't even exist.

This is some abstract item type that should eventually just be refactor into being a *proper* radio, but that's beyond the scope of this PR, and there's enough differences that correcting them would be no easy feat in a short time frame.

**Comment by Kyep** [comment]
This PR is OK, as it is just a refactor, and we should generally trust the maints when it comes to what refactors are a good idea for code quality.

That said, I'm a little uncomfortable with the process by which this PR came to be. I'm always uncomfortable with us seeming to do something primarily because "TG did it". I'm even more uncomfortable with someone pushing something on TG, then using "TG did it" as an excuse to speedmerge that same thing here without *our players* having proper time to object to it.

In *this case*, though, I can see why this weird process is OK. This PR is a refactor, so it won't affect player experience, so players have no reason to object to it. So, its fine to use this process in *this case*. Nobody is going to object, anyway.

Well, nobody except all the other PR authors whose PRs, despite being made earlier, are now going to conflict with a bigger PR that was made later and merged more quickly. That's a PITA for everyone who has open PRs. Still, I have to assume that the maints care enough about not alienating PR authors that they wouldn't do this without a good reason.

The main thing I'd like to note here is that this sort of process is OK in *this case*, but wouldn't be acceptable for, say, a feature PR. It would feel like our community is being cut out of the loop with regards to their ability to object to a PR before it is merged. For a refactor like this I guess its OK, but only for a refactor like this. Feature PRs should never use this process.

**Comment by KasparoVy** [comment]
'tis why I no longer bother

**Comment by FreeStylaLT** [comment]
I HAD A HUNCH THAT I SHOULDN'T FIX THOSE CONFLICTS YET

**Comment by FreeStylaLT** [comment]
AND LOOK AT THAT
A WEEK LATER MISTER FOXMAN BURSTS OUT OF THE BUSHES WITH ANOTHER SHANK FOR EVERY PR OUT THERE

**Comment by Fox-McCloud** [comment]
@alexkar598 Be nice. Map conflicts *are* a pain in the ass.

**Pull request merged**